docs/source/commands/peerlifecycle.md

```diff
@@ -361,11 +361,13 @@ A chaincode needs to be packaged before it can be installed on your peers.
 This example uses the `peer lifecycle chaincode package` command to package
 a Go chaincode.
 
+  * Use the `--path` flag to indicate the location of the chaincode.
```

Comment:
FYI: you need to make these changes in docs/wrappers/peer_lifecycle_chaincode_postscript.md and then run "make clean help-docs". Otherwise, the next time somebody submits a PR with an updated command reference this change will be lost.

Reply:
@wlahti That's how I did it :-)
You'll see that both files are updated in this PR.

Reply:
Oops, guess I was reviewing before my brain fully woke up. Carry on. :)